app/org/sagebionetworks/bridge/dynamodb/DynamoSurvey.java

```diff
@@ -71,8 +72,9 @@ public DynamoSurvey(DynamoSurvey survey) {
     }
     
     @Override
-    @DynamoDBAttribute(attributeName = "studyKey")
     @JsonIgnore
+    @DynamoDBAttribute(attributeName = "studyKey")
+    @DynamoDBIndexHashKey(globalSecondaryIndexName = "studyKey-index")
```

Comment:
DynamoInitializer currently only creates global secondary indices with Projection.KEYS_ONLY. Your code assumes Projection.ALL. This is probably not a problem if we're creating indices manually, but it will be a problem if we launch new stacks or bootstrap new team members.

If we're not fixing it now, we should create a JIRA item and document affected tables so future us will know.